arangod/VocBase/LogicalDataSource.cpp

```diff
@@ -54,6 +57,7 @@ std::string ensureGuid(
     TRI_ASSERT(planId);
     guid.append("c");
     guid.append(std::to_string(planId));
+    guid.push_back('/');
```

Comment:
Why does the guid have to end with a slash here?
A comment would make sense IMHO.